### PR TITLE
refactor: image optimisations and safer imageBaseUrl

### PR DESCRIPTION
**Pull request by benw202**
## Description

This PR optimises all images using ShortPixel

It also makes the `imageBaseUrl` 'safer' by removing the trailing `/` if present. This is done in `constants.ts`. The command itself now has the `/` in it.

`.env.example` now contains the actual URL's used

## Test Results

N/A

## Discord Username

BenW#8484

### CODE REVIEW

File: assets/images/a32nx/adirs.png
Comment (pdellaert) [inline assets/images/a32nx/adirs.png:1]: Does it need to be that big, maybe we should half the resolution as well?

File: assets/images/support/dlss.png
Comment (pdellaert) [inline assets/images/support/dlss.png:1]: Same thing here... do we need that resolution?

**Review by pdellaert — APPROVED** [review]
Just two questions on image scale, but all good!

File: assets/images/a32nx/adirs.png
Comment (benw202) [reply assets/images/a32nx/adirs.png:1]: Resolution has been halved

File: assets/images/support/dlss.png
Comment (benw202) [reply assets/images/support/dlss.png:1]: Resolution has been halved

**Review by oim1 — APPROVED** [review]
all files look good

**Review by pdellaert — APPROVED** [review]

**Review by pdellaert — APPROVED** [review]

**Pull request merged**